spec/mailers/activity_mailer_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe ActivityMailer, type: :mailer do
-  let(:template_path) { File.join(__dir__, '../fixtures/activity_mailer/test_email') }
+  let(:template_path) { '../../spec/fixtures/activity_mailer/test_email' }
```

Comment:
In my code, I've used `options[:template_path] = File.dirname(template)` to set the path of template. Although I prepend a path to a mailer, it will still find the template from `dirname(template)`. Which means if I prepend a path first, and just input a template name in spec, it will find the template from `'/'` and the test will fail.